index.js

```diff
@@ -29,7 +29,10 @@ const removeMootSeparators = (string, separator) => {
 };
 
 module.exports = (string, options) => {
-	if (typeof string !== 'string') {
+	if (typeof string === "number") {
+		string = string.toString(); // my filename may be a int or float number
```

Comment:
I'm not interested in silently coercing unrelated types. This method accepts a string, if you want it to magically support other things you can do the coercion yourself when calling this method.